staking/contracts/contracts.py

```diff
@@ -3,6 +3,8 @@
 import json
 from pyteal import *
```

Comment:
In the context of smart contract development, I recommend against * imports. Rather, every external object should be explicitly imported as this is the safest approach.